Tests/NIOHTTPClientTests/SwiftNIOHTTPTests.swift

```diff
@@ -170,6 +170,32 @@ class SwiftHTTPTests: XCTestCase {
         response = try httpClient.get(url: "http://localhost:\(httpBin.port)/redirect/https?port=\(httpsBin.port)").wait()
         XCTAssertEqual(response.status, .ok)
     }
+    
+    func testHttpHostRedirect() throws {
+        let httpClient = HTTPClient(eventLoopGroupProvider: .createNew,
+                                    configuration: HTTPClient.Configuration(certificateVerification: .none, followRedirects: true))
+        
+        defer {
+            try! httpClient.syncShutdown()
+        }
+        
+        let response = try httpClient.get(url: "https://httpbin.org/redirect-to?url=https%3A%2F%2Fwww.httpbin.org%2Fheaders").wait()
+        guard let body = response.body else {
+            XCTFail("The target page should have a body containing request headers")
+            return
+        }
+        let data = body.withUnsafeReadableBytes {
+            Data(bytes: $0.baseAddress!, count: $0.count)
+        }
+        
+        let decoder = JSONDecoder()
+        struct HeadersContainer: Decodable {
+            var headers: [String:String]
```

Comment:
small formatting issue: missing space after semicolon

Reply:
lets get https://github.com/swift-server/swift-nio-http-client/pull/23 and run `swiftformat`

Reply:
@artemredkin @tomerd  Ran `swiftformat` in the latest commit. Thanks